ctapipe/image/extractor.py

```diff
@@ -21,48 +21,60 @@
 import numpy as np
 from traitlets import Int
 from ctapipe.core import Component
-from numba import njit, prange, float64, float32, int64
-
-
-def extract_charge_from_peakpos_array(waveforms, peakpos, width, shift):
+from numba import njit, prange, guvectorize, float64, float32, int64
+
+
+@guvectorize(
+    [
+        (float64[:], int64, int64, int64, float64[:]),
+        (float32[:], int64, int64, int64, float64[:]),
+    ],
+    '(s),(),(),()->()',
+    nopython=True,
+)
+def sum_samples_around_peak(waveforms, peak_index, width, shift, ret):
     """
     Sum the samples from the waveform using the window defined by a
-    peak postion, window width, and window shift.
+    peak position, window width, and window shift.
+
+    This function is a numpy universal function which defines the operation
+    applied on the waveform for every channel and pixel. Therefore in the
+    code body of this function:
+        - waveforms is a 1D array of size n_samples.
+        - peak_index, width and shift are integers, corresponding to the
+            correct value for the current pixel
+
+    The ret argument is required by numpy to creae the numpy array which is
```

Comment:
typo in creae

Reply:
Fixing this tiny typo results in all reviews requiring restoration, and all test being ran. Annoying!

Reply:
yes, is there somthing smarter? Probably not, right?

Reply:
Maybe we should just deactivate the automatic dismissal of reviews and just do it by hand if the author think changes require new reviews.